AmplifyPlugins/Auth/Tests/AuthHostApp/AuthWatchApp.entitlements

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>keychain-access-groups</key>
+	<array>
```

Comment:
is this change part of the PR or for internal testing?

Reply:
Part of the PR (same reasoning as before)